lldb/test/Shell/SwiftREPL/RedirectInputUnreadable.test

```diff
@@ -9,3 +9,4 @@
 
 < A.swift
```

Comment:
You'd want to change this, too, right?

Reply:
I thought `%t` was only substituted in comments, does it apply to all text in the file?

Reply:
you are right, of course. The A.swift here is already implicitly remapped because of the RUN: cd %t above.